### PR TITLE
3 module

### PR DESCRIPTION
**Pull request by JavohirMir**

### CODE REVIEW

File: src/components/DeleteMovieModal/index.tsx [hunk 1]
```diff
@@ -0,0 +1,26 @@
...
+        <Typography id="edit-movie-modal-title" variant="h2">
+          Delete Movie
+        </Typography>
+        <Box id="edit-movie-modal-description">
+          <Typography sx={{ marginTop: "10px" }}>
+            Are you sure you want to delete this movie?
+          </Typography>
+          <Box sx={{ display: "flex", justifyContent: "flex-end" }}>
+            <Button sx={{ margin: "10px" }} size="large" variant="contained">
+              CONFIRM
```
Comment (khusanov-95) [inline src/components/DeleteMovieModal/index.tsx]: lower case => make it upper case with css

File: src/pages/HomePage/index.tsx [hunk 2]
```diff
@@ -1,20 +1,92 @@
...
+    );
+    if (this.state.sortOption === "release date") this.sortByDate(sortedMovies);
+    else if (this.state.sortOption === "duration")
+      this.sortByDuration(sortedMovies);
+    else this.sortByRating(sortedMovies);
+  };
+
+  sortByRating = (movies: Movie[]) => {
+    this.setState({
+      movies: movies.sort((a, b) => +b.imdbRating - Number(a.imdbRating)),
```
Comment (khusanov-95) [inline src/pages/HomePage/index.tsx]: we probably need a comment here with explanation, also why both "+" and Number() used ?

Comment (JavohirMirzaakbarov) [reply src/pages/HomePage/index.tsx]: Fixed! Now "+" used for converting string rating into number!

**Comment by JavohirMirzaakbarov** [comment]
@khusanov-95 I have fixed the issues you have mentioned, please approve to merge if everything is Ok now.

**Pull request merged**